ui/admin/tests/acceptance/targets/create-test.js

```diff
@@ -116,43 +117,49 @@ module('Acceptance | targets | create', function (hooks) {
     const tcpTargetCount = getTCPTargetCount();
     await visit(urls.targets);
 
-    await click(`[href="${urls.newTarget}"]`);
-    await click('[value="tcp"]');
-    await fillIn('[name="name"]', 'random string');
-    await click(SAVE_BTN_SELECTOR);
+    await click(commonSelectors.HREF(urls.newTarget));
+    await click(selectors.FIELD_TYPE_VALUE('tcp'));
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
+    await click(commonSelectors.SAVE_BTN);
 
     assert.strictEqual(getTargetCount(), targetCount + 1);
     assert.strictEqual(getTCPTargetCount(), tcpTargetCount + 1);
     assert.strictEqual(
       this.server.schema.targets.all().models[getTargetCount() - 1].name,
-      'random string',
+      commonSelectors.FIELD_NAME_VALUE,
     );
   });
 
   test('default port is not marked required for SSH targets', async function (assert) {
     featuresService.enable('ssh-target');
-    await visit(urls.newTarget);
-    assert.dom('[data-test-default-port-label]').includesText('Optional');
+    await visit(urls.targets);
+
+    await click(commonSelectors.HREF(urls.newTarget));
+
+    assert.dom(selectors.FIELD_DEFAULT_PORT_LABEL).includesText('Optional');
   });
 
   test('default port is marked required for TCP targets', async function (assert) {
     featuresService.enable('ssh-target');
-    await visit(urls.newTarget);
-    await click('[value="tcp"]');
-    assert.dom('[data-test-default-port-label]').includesText('Required');
+    await visit(urls.targets);
+
+    await click(commonSelectors.HREF(urls.newTarget));
+    await click(selectors.FIELD_TYPE_VALUE('tcp'));
+
+    assert.dom(selectors.FIELD_DEFAULT_PORT_LABEL).includesText('Required');
```

Comment:
Nice work.

Non-blocking: I noticed when mirage is enabled and you modify a target `type=tcp` by removing the required port value - you can still save the form. Not an issue in prod, just raising here (if the goal is to have mirage behave as closely to prod as possible).

A test that modifies an existing target `type=tcp` by removing the required port val and making sure the form cannot save might be a good catch but just a non-blocking suggestion.

Reply:
In prod, you'll get this err:

<img width="1428" alt="Screenshot 2025-05-13 at 2 34 54 PM" src="https://github.com/user-attachments/assets/f3e59c2b-6380-4247-8879-7eef0f2406c2" />

Reply:
Good eye! Currently in our mirage route handlers we don't do error validation. We do mock out the error responses in acceptance test for all resources tho (usually in the update-test.js file). [Here's an example](https://github.com/hashicorp/boundary-ui/pull/2824/files#diff-c5e5d0925a43d57816f07a30c5bcc79613e021a70ffda6d680d696a1f7264601R94). I'll add one for the `default port` field as well!